data/Identifying_production_data_types.md

```diff
@@ -1,3 +1,216 @@
 # Data types found in production service repos
 
+---
 
+#Expression
+##Description this stores expression data of a variety of different types (microarray, RNA-Seq, Proteomics, q-PCR)
+
+##Methods
```

Comment:
Let's break this section on methods out into another document.  I just want to capture the current state of data types here.